src/Core/src/Handlers/ScrollView/ScrollViewHandler.iOS.cs

```diff
@@ -387,6 +401,9 @@ public void Disconnect(UIScrollView platformView)
 
 				platformView.Scrolled -= Scrolled;
 				platformView.ScrollAnimationEnded -= ScrollAnimationEnded;
+
+				if (platformView is MauiScrollView mauiScrollView)
```

Comment:
This is tricky, the Disconnect might not be called sometimes.

Reply:
Mnn, what is the scenario? `ConnectHandler` and `DisconnectHandler` are the most used approach and the recommended  way in the documentation. We would have to cover the scenarios where they do not work as expected to avoid it.

What would be the correct approach to ensure the unsubscription?, Detect property changes and check when `Window` is null?

Reply:
Take a look to the MauiScrollView and use a WeakReference to manage the layout event. Similar to: https://github.com/dotnet/maui/pull/15946

Reply:
Updated, could you review it again Rui?

Reply:
ScrollView is tested here:
https://github.com/dotnet/maui/blob/412eb059d825447480d671f9d5d065c992b71ed0/src/Controls/tests/DeviceTests/Memory/MemoryTests.cs#L46

The new event looks the same as the existing ones inside `ScrollEventProxy`, so this looks correct.